.eleventy.js

```diff
@@ -85,6 +85,9 @@ module.exports = function(eleventyConfig) {
         "src/public/": "/",
     });
 
+    // Copy the contents of the `_redirects` to the output folder
+    eleventyConfig.addPassthroughCopy("src/_redirects");
```

Comment:
Don't think this is needed if we keep the redirect.njk?

Reply:
yes, i have removed it